README.md

```diff
@@ -71,4 +79,9 @@ The Sir Syed Global Scholar Award (SSGSA) is annually administered by the Sir Sy
       ```
         git push origin <branch name>
       ```
-   - Create a pull request to this repository and mention the issue (if required)
+   - Create a pull request to this repository and mention the respective github issue.
```

Comment:
Add a point that says creating a PR is mandatory, even for the smallest possible change.